src/sass/components/_position.scss

```diff
@@ -1,7 +1,59 @@
+// Position Item
+
+@include media-object('.position-item', '.position-item__avatar', '.position-item__content');
+
 .position-item {
-  .position-source {
-    color: rgba(171,171,171, 1);
-    font-size: 0.9em;
-    font-weight: 200;
-  }
+  position: relative;
```

Comment:
@pertrai1 Can you be more specific? I'm trying to link names to the components they style. I know there is still some refactoring to be done, but I'm not clear what 'this many responsibilities' refers to.